src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -82,15 +82,35 @@ const getRelationParts = (val: unknown): string[] => {
   return []
 }
 
+const getWdsUrl = apps => {
```

Comment:
Should this logic be tested?

Reply:
agreed this is a strong candidate for a test, now that we have time to circle back and address it